### PR TITLE
adds simplecov gem

### PR DESCRIPTION
**Pull request by jtruong2**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,6 +28,7 @@ group :development, :test do
   gem 'shoulda-matchers'
   gem 'faker'
   gem 'selenium-webdriver'
+  gem 'simplecov'
```
Comment (swdonovan) [inline Gemfile]: Good habits that we're getting into!

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
+require 'simplecov'
+SimpleCov.start 'rails'
```
Comment (swdonovan) [inline spec/rails_helper.rb]: no copying and pasting

Comment (wratterman) [inline spec/rails_helper.rb]: @jtruong2 Good catch adding 'rails' after `.start`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+  - 2.2
+  - 2.3.4
+  - 2.4.1
```
Comment (wratterman) [inline .travis.yml]: @markyv18 Will you check to make sure that your ruby version is included here?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  - jruby
```
Comment (erose357) [inline .travis.yml]: I think we can get rid of jrby and any versions before 2.3.

**Comment by wratterman** [comment]
I am seeing travis.cl is passing, this is good to merge and we all approve. Sounds good to merge! 👍

**Pull request merged**